rewrite-xml/src/main/java/org/openrewrite/xml/trait/SpringReference.java

```diff
@@ -62,6 +65,21 @@ public boolean supportsRename() {
         return true;
     }
 
+    @Override
+    public Tree rename(Renamer rename, Cursor cursor, ExecutionContext ctx) {
```

Comment:
I think the name of this function is still a bit confusing, as this function does not make any modification to the references itself, nor does it return a new reference, effectively, the method can be static, as the cursor is used.